src/AssemblyAI/Files/FilesClient.cs

```diff
@@ -28,8 +33,9 @@ public async Task<UploadedFile> UploadAsync(Stream request, RequestOptions? opti
                 Method = HttpMethod.Post,
                 Path = "v2/upload",
                 Body = request,
-                Options = options
-            }
+                Options = options,
+            },
+            cancellationToken
         );
         var responseBody = await response.Raw.Content.ReadAsStringAsync();
```

Comment:
`cancellationToken` should be passed to `response.Raw.Content.ReadAsStringAsync` too

Reply:
Anything that's async usually

Reply:
Also `.ConfigureAwait(false)`

Reply:
You'll need to use conditional compilation tho:
```csharp
#if NET6_0_OR_GREATER
        var responseBody = await response.Raw.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
#else
        var responseBody = await response.Raw.Content.ReadAsStringAsync().ConfigureAwait(false);
#endif
```